composer.json

```diff
@@ -19,9 +19,9 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder":           "~2.3|~3.0",
-        "symfony/validator":        "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.1",
```

Comment:
@akovalyov Symfony have two versions. LTS (2.8) and latest feature (3.1). Why we need to support old 3.0 versions?
http://symfony.com/download